src/test/modules/Execution.test.js

```diff
@@ -46,13 +46,13 @@ var parserActorssArray = [
             'xScope': 10,
             'yScope': 10
         }, 'position': {
-            'xPos': 0,
+            'xPos': 10,
             'yPos': 15
         }}];
 
 const testExecution = new Execution ();
-test('The execution of scenario need to return the actual status of the actors instances after actions being executed', () => {
-    expect(testExecution.execute(parserActionssArray, parserActorssArray, 18)).toEqual([
+test('The execution of scenario need to return the actual status of the actors instances after actions being executed, all actions are executed after 5 scene, using a minor value in Execution scenes attributes will return an unexpected value in health for RAT1', () => {
```

Comment:
Isn't this a little to long?

Reply:
Yes it is :( , but as i mentioned before it's the descriptable explaination of that specific scenario and how it works, to avoid doubts about how or why, the idea is making explicable by itself.

"all actions are executed after 5 scene, using a minor value in Execution scenes attributes will return an unexpected value in health for RAT1"